### ISSUE TITLE
Calling Trader.InitAccount with insufficient funds results in a pending open account which can't be closed

### ISSUE THREAD

**Issue by cwispp** [lightninglabs/pool#409]
Calling Trader.InitAccount with insufficient funds results in a pending open account which can't be closed with Trader.CloseAccount. Adding more funds using Trader.DepositAccount also doesn't work.

**Comment by guggero** [comment]
Thanks for the report! Yeah, it looks like the check whether the wallet has enough balance comes too late, after the account was already reserved on the server and written to the database.
We want to improve the account funding anyway (switching over to using the PSBT APIs of `lnd`). Also related: https://github.com/lightninglabs/pool/issues/37. So we'll make sure to add that check earlier as well when doing the refactor.

**Comment by paboum** [comment]
Having the same issue still after 4 months. Can't either run `litd` now or log in to UI. I'm revoking RPC access since this is a blocker, investing my liquidity on uniswap or whatever.

**Comment by guggero** [comment]
Yes, unfortunately we don't have a solution for this yet. You can close your account and get your funds out with [`chantools closepoolaccount`](https://github.com/guggero/chantools/blob/master/doc/chantools_closepoolaccount.md) though.
